tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -3541,7 +3541,8 @@ port::StatusOr<std::unique_ptr<cudnn_frontend::OperationGraph>>
 GetCudnnFusedOperationGraph(
     dnn::ConvolutionKind kind, dnn::DataType input_type,
     dnn::DataType bias_type, dnn::DataType output_type, double alpha,
-    double alpha2, const dnn::BatchDescriptor& input_descriptor,
+    double alpha2, double leakyrelu_alpha,
+    const dnn::BatchDescriptor& input_descriptor,
     const dnn::FilterDescriptor& filter_descriptor,
     dnn::BatchDescriptor bias_descriptor,
     const dnn::BatchDescriptor& output_descriptor,
```

Comment:
based on the code and comments below, it seems that the comment on line 3555 (starting with "CUDNN fused operation supports the pattern"...) needs to be updated.